core-bundles/language/org.eclipse.vorto.editor.datatype/src/org/eclipse/vorto/editor/datatype/Datatype.xtext

```diff
@@ -27,12 +28,16 @@ QualifiedName:
 	ID ('.' ID)*
 ;
 
+VortolangVersion:
```

Comment:
Am I reading it wrong that VortolangVersion is only used in the enum and otherwise it's defined as FLOAT? Shouldn't this be removed and also declared as FLOAT in the enum?

Reply:
That is one of the reasons why the build isn't working - the other one being the rule is still there in Datatype.xtext :)
Looks like the merge from dev was more messed up than I thought. 
Am fixing locally and will push again.

Reply:
Build is green now. Really not sure what happened SCM-wise but the right change set is now in after my last push.